{{ cookiecutter.package_name }}/{{ cookiecutter.module_name }}/conftest.py

```diff
@@ -5,7 +5,13 @@
 from astropy.tests.pytest_plugins import *
 
 ## Uncomment the following line to treat all DeprecationWarnings as
-## exceptions
+## exceptions. For Astropy v2.0 or later, there are 2 additional keywords,
+## as follow.
+## To ignore some modules that produce deprecation warnings on import, add:
+##     modules_to_ignore_on_import=['module_1', 'module_2']
```

Comment:
I guess you are right as far as user is concerned although technically it can be either package or module. Sure, I'll list them.